app/controllers/users_controller.rb

```diff
@@ -40,7 +40,7 @@ def index
 
   def update
     raise Pundit::NotAuthorizedError if current_user.organisation_admin? &&
-      ! current_user.organisation.subtree.map(&:id).include?(params[:user][:organisation_id].to_i)
+                                        ! current_user.organisation.subtree.map(&:id).include?(params[:user][:organisation_id].to_i)
```

Comment:
This indentation looks excessive; can this Rubocop behaviour be disabled or tweaked?